libpcsxcore/misc.c

```diff
@@ -169,6 +171,45 @@ static void fake_bios_gpu_setup(void)
 		GPU_writeData(gpu_data_def[i]);
 }
 
+/* Function for automatic patching according to GameID.
+ * It's possible that some of these games have no IDs, like some japanese games i encountered.
+ * I need to check whenever this matters or not for our games.
+ * (Plus it can still be activated in the menu)
+ * Let's hope the IDs are also not shared with other games ! (Homebrew, don't screw it up)
+ * */
+static void CheckforCDROMid_applyhacks()
+{
+	uint32_t i;
+
+	/* Apply hack battle fix for Inuyasha - Sengoku Otogi Kassen */
+	if (strncmp(CdromId, "SLPS03503", 9) == 0)
+	{
```

Comment:
With this we get some of the IDs here and some in database.c... Maybe just put them all here in misc.c .